### PR TITLE
Update 3.0.0

### PR DESCRIPTION
**Pull request by OmniacDev**
Update the template to use new formats for version 3.0.0 of Amethyst Launcher

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
-set(MOD_AUTHOR "FrederoxDev")
+####################################################################################################
+set(SHARD_NAME "Shard Template")
+set(SHARD_AUTHOR "Author")
+set(SHARD_UUID "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX")
+set(SHARD_VERSION "0.0.0")
+
+# ADVANCED
+set(SHARD_FORMAT "MOD")
+set(SHARD_FORMAT_VERSION "3.0.0") # Don't modify
```
Comment (FrederoxDev) [inline CMakeLists.txt:9]: Shard is just the file extension, continue referring to these as mods

**Review by FrederoxDev — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (OmniacDev) [reply CMakeLists.txt:9]: they aren't always mods though, and calling em shards aint gonna make much of a difference

Comment (RestartFU) [reply CMakeLists.txt:9]: > they aren't always mods though, and calling em shards aint gonna make much of a difference 

bro who are you mad at 😂

Comment (OmniacDev) [reply CMakeLists.txt:9]: > > they aren't always mods though, and calling em shards aint gonna make much of a difference
> 
> bro who are you mad at 😂

no one? i'm just replying

Comment (FrederoxDev) [reply CMakeLists.txt:9]: > they aren't always mods though, and calling em shards aint gonna make much of a difference 

Please stop giving so much resistance to everything, it's pointless. We are not renaming the word mod

Comment (OmniacDev) [reply CMakeLists.txt:9]: I'm not being resistant for no reason. Keeping the name as "Mod" is confusing;
- why have the format field? A mod "mod"? a mod "runtime"?
- why is the file name .shard but they're called mods?

Plus changing the variable names literally ain't gonna do anything. You can still can em mods if you want, but the actual naming has to be clear what it is. It's not specifically a mod.

**Pull request closed without merging**